wordpress-mobile/WordPress-iOS
# UL&S: InstructionLabelTableViewCell added

Fixes #n/a

Ref. PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/305

This is a test PR for the UL&S Authenticator changes made to the SiteAddressViewController UI. 

### To test:
1. `rake dependencies` and ensure no WordPressKit errors appear
2. Build and run
3. Log out if logged in
4. Log in > "Or log in by entering your site address"

<a href="https://user-images.githubusercontent.com/1062444/84803736-ff9c2f80-afc7-11ea-9f71-c4a6acf4384c.png"><img src="https://user-images.githubusercontent.com/1062444/84803736-ff9c2f80-afc7-11ea-9f71-c4a6acf4384c.png" width="350" /></a>

### Do not merge until:
1. Feature flag is disabled for `.unifiedSiteAddress`
2. The Authenticator PR has been merged: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/305
3. A new Authenticator release has been created
4. The release is published on Cocoapods trunk
5. This PR points to the new release

PR submission checklist:
